src/components/textarea/textarea.tsx

```diff
@@ -0,0 +1,411 @@
+import {
+  Component,
+  h,
+  Prop,
+  Element,
+  Event,
+  EventEmitter,
+  VNode,
+  Watch,
+  Method,
+  Host,
+  State
+} from "@stencil/core";
+import { connectForm, disconnectForm, FormComponent, HiddenFormInputSlot } from "../../utils/form";
+import { connectLabel, disconnectLabel, getLabelText, LabelableComponent } from "../../utils/label";
+import { slotChangeGetAssignedElements } from "../../utils/dom";
+import { CSS, SLOTS, RESIZE_TIMEOUT } from "./resources";
+import {
+  connectLocalized,
+  disconnectLocalized,
+  LocalizedComponent,
+  NumberingSystem,
+  numberStringFormatter
+} from "../../utils/locale";
+import { createObserver } from "../../utils/observers";
+import {
+  componentLoaded,
+  LoadableComponent,
+  setComponentLoaded,
+  setUpLoadableComponent
+} from "../../utils/loadable";
+import {
+  connectMessages,
+  disconnectMessages,
+  setUpMessages,
+  T9nComponent,
+  updateMessages
+} from "../../utils/t9n";
+import { TextareaMessages } from "./assets/textarea/t9n";
+import { throttle } from "lodash-es";
+
+/**
+ * @slot - A slot for adding text.
+ * @slot footer-leading - A slot for adding a leading footer.
+ * @slot footer-trailing - A slot for adding a trailing footer.
+ */
+
+@Component({
+  tag: "calcite-textarea",
+  styleUrl: "textarea.scss",
+  shadow: true,
+  assetsDirs: ["assets"]
+})
+export class Textarea
+  implements FormComponent, LabelableComponent, LocalizedComponent, LoadableComponent, T9nComponent
+{
+  //--------------------------------------------------------------------------
+  //
+  //  Element
+  //
+  //--------------------------------------------------------------------------
+  @Element() el: HTMLCalciteTextareaElement;
+
+  //--------------------------------------------------------------------------
+  //
+  //  Properties
+  //
+  //--------------------------------------------------------------------------
+
+  /** When `true`, focuses the `textarea` element on page render. */
+  @Prop({ reflect: true }) autofocus = false;
```

Comment:
Can you change the comment to match `calcite-input`:
```js
  /**
   * When `true`, the component is focused on page load.
   *
   * @mdn [autofocus](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/autofocus)
   */
```

Can you also add the `@mdn` jsdoc tag where applicable? See `calcite-input` for examples. We plan to eventually add it as a chip/link  in the documentation site